modules/@apostrophecms/schema/ui/apos/components/AposArrayEditor.vue

```diff
@@ -329,4 +363,30 @@ export default {
   .apos-modal-array-min-error, .apos-modal-array-max-error {
     color: var(--a-danger);
   }
+
+  .apos-modal-array-items {
+    margin: 10px;
+  }
+
+  .apos-modal-array-items__heading {
+    position: relative;
+    margin: 20px 4px;
+  }
+
+  // Specificity needed due to AposButton rules
+  .apos-modal-array-items__heading .apos-modal-array-items__add {
+    position: absolute;
+    top: -5px;
+    right: 0;
+  }
+
+  .apos-modal-array-items__label {
+    // Consistent with appearance of same information in input field
+    letter-spacing: 1.5px;
```

Comment:
I don't like that we have so many "bespoke" text styles, but this didn't create that problem, so proceed...